src/credentials/CHIPCert.h

```diff
@@ -432,10 +442,12 @@ class DLL_EXPORT ChipCertificateSet
      *
      * @param reader       A TLVReader positioned at the CHIP certificate TLV structure.
      * @param decodeFlags  Certificate decoding option flags.
+     * @param chipCert     Buffer containing certificate encoded on CHIP format.
+     * @param chipCertLen  The length of the certificate buffer.
```

Comment:
This is where we could use lifetime documentation: How long does this buffer need to stay valid?  Can it be freed after this function returns?

Reply:
It needs to stay valid for as long as the certificate data is used, comments have been updated to reflect this.